libraries/libfc/include/fc/variant.hpp

```diff
@@ -694,6 +695,35 @@ namespace fc
       n = boost::multiprecision::number<T>(v.get_string());
    }
 
+   template<typename T> void to_variant( const boost::dynamic_bitset<T>& bs, fc::variant& v ) {
+      auto num_blocks = bs.num_blocks();
+      if( num_blocks > MAX_NUM_ARRAY_ELEMENTS ) throw std::range_error( "too large" );
+
+      std::vector<uint32_t> blocks(num_blocks);
+      boost::to_block_range(bs, blocks.begin());
+
+      std::vector<fc::variant> vars;
+      vars.reserve(num_blocks);
+      for( const auto& b: blocks ) {
+         vars.push_back( fc::variant(b) );
+      }
+      v = std::move(vars);
+   }
+
+   template<typename T> void from_variant( const fc::variant& v, boost::dynamic_bitset<T>& bs ) {
+      const std::vector<fc::variant>& vars = v.get_array();
+      auto num_vars = vars.size();
+      if( num_vars > MAX_NUM_ARRAY_ELEMENTS ) throw std::range_error( "too large" );
+
+      std::vector<uint32_t> blocks;
+      blocks.reserve(num_vars);
+      for( const auto& var: vars ) {
+         blocks.push_back( var.as<uint32_t>() );
+      }
+
+      bs = { blocks.cbegin(), blocks.cend() };
+   }
```

Comment:
Would like to see these moved out of variant.hpp and put in a `fc/dynamic_bitset.hpp`